### PR TITLE
NW6 | Rabia Avci | Full-Stack-Project | Week-2 |  New video form 

### PR DESCRIPTION
**Pull request by RbAvci**
Completed these tasks for this ticket:

- Added a user-friendly form for capturing new video recommendations.
- The form supports fields for video titles and YouTube URLs.
- Fetch and save to database is done.
- Add CSS for new form component

### CODE REVIEW

File: client/src/NewVideoForm.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+button {
+	padding: 10px 20px;
+	border: none;
+	border-radius: 5px;
+	background-color: #007bff;
+	color: #fff;
+	font-size: 16px;
+	cursor: pointer;
```
Comment (Ara225) [inline client/src/NewVideoForm.css:49]: Is there any particular reason you're setting  the cursor to pointer here? It should be pointer by default here https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

File: client/src/NewVideoForm.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	transition: background-color 0.3s;
+}
+
+button:hover {
+	background-color: #0056b3;
```
Comment (Ara225) [inline client/src/NewVideoForm.css:54]: Always nice to have a hover style for buttons. 

Overall, I really like what you're doing here, good use of flex and nice touches like border-radius and padding in inputs throughtout

File: client/src/NewVideoForm.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+		e.preventDefault();
+
+		fetch("/api/videos", {
+			method: "POST",
+			headers: {
+				Accept: "application/json",
+				"Content-Type": "application/json",
+			},
+			body: JSON.stringify({
+				title: e.target.title.value,
```
Comment (Ara225) [inline client/src/NewVideoForm.jsx:13]: This is a perfectly acceptable way of handling inputs, especially for a small form, however it's  worth being aware that there's a different (not necessarily better, it just serves a different purpose) way of doing this that you may need someday

https://react.dev/reference/react-dom/components/input#controlling-an-input-with-a-state-variable

**Review by Ara225 — COMMENTED** [review]
All good, nothing to complain about. Good work :)

File: client/src/NewVideoForm.css (hunk 1 above)
Comment (RbAvci) [reply client/src/NewVideoForm.css:49]: When its default it keeps an arrow, i wanted to make it a hand icon when hovering over the element :)

**Review by zelihapala — APPROVED** [review]

**Review by zelihapala — APPROVED** [review]

**Pull request merged**